File: official/docs/php/v6/shipping-refund/refund.php
<?php

$client = new \EasyPost\EasyPostClient(getenv('EASYPOST_API_KEY'));

$shipment = $client->shipment->retrieve('shp_...');

$refundedShipment = $client->shipment->refund($shipment->id);

echo $refundedShipment;

File: official/docs/golang/current/insurance/refund.go
package example

import (
	"fmt"

	"github.com/EasyPost/easypost-go/v4"
)

func refund() {
	client := easypost.New("EASYPOST_API_KEY")

	insurance, _ := client.RefundInsurance("ins_...")

	fmt.Println(insurance)
}
